woocommerce/woocommerce-android
# Dark Mode: Main Settings Screen

Partially fixes #1584 by implementing light/dark styling to the main settings screen. 

### Highlights
* Created a new custom component called `WCSettingsOptionValueView` to simplify working with settings views and styling those views.
* Added new setting styles and applicable dimensions. 

![settings-light](https://user-images.githubusercontent.com/5810477/72403606-fbfe3700-370f-11ea-923f-3913ce70cb9d.png)

![settings-dark](https://user-images.githubusercontent.com/5810477/72403609-fdc7fa80-370f-11ea-8ad9-d36a69898fad.png)

### Not Included
- The color of the switch option will be changed in a different PR when switch theming is implemented. 
- Dialogs or child pages reached when clicking on a setting option in the main screen.

### To Test
- Test on a devices running < 26  and over 26 to verify notification options designs. 
- Be sure to click through on options and verify they act and update as expected.
- Verify in light and dark modes.
- Verify landscape

### Design
@kyleaparker: 
* There were some options missing from the designs such as “Feature Request” and “Image Optimization” - so I just guessed on how you wanted them laid out. Please verify.
* I kept the “LOG OUT” button text in all caps to follow styling with the rest of the app. 
* With the new alignment with the appbar title
